### ISSUE TITLE
Problem with macforwardingconfigurator 

### ISSUE THREAD

**Issue by lukefadedaway** [omnetpp/omnetpp#1058]
**Platform**
 - OMNeT++ version: 6.0.1
 - OS: Ubuntu 22.04
 - INET version: 4.5.2

Sending packets to the first Host Device in the NED file leads to the packet being handled by the switch like a broadcast message. 
I encountered this bug when setting up a TSN Network and not manually setting up the MAC Forwarding tables. 
This bug should've been fixed according to this issue: https://github.com/inet-framework/inet/issues/880 , but it's still happening. 

To reproduce this bug, use these two files: 
(Configuration A-to-B will run without the bug because B is not the first Device in the ini file. B-to-A will show the bug because A is the first device and therefore can't bee addressed)
bugtest.ini :
```
[A-to-B]
network = bugtest

*.*.eth[*].bitrate = 1Gbps

*.A.numApps = 1
*.A.app[0].typename = "UdpSourceApp"
*.A.app[0].display-name = "A-B"
*.A.app[0].io.destAddress = "B"
*.A.app[0].io.destPort = 1000 
*.A.app[*].source.packetNameFormat = "%M-%m-%c"
*.A.app[0].source.packetLength = 1200B 
*.A.app[0].source.productionInterval = 490000ns  

# server applications
*.B.numApps = 1
*.B.app[*].typename = "UdpSinkApp"
*.B.app[0].display-name = "A-B"
*.B.app[0].io.localPort = 1000 

[B-to-A]
network = bugtest

*.*.eth[*].bitrate = 1Gbps

*.B.numApps = 1
*.B.app[0].typename = "UdpSourceApp"
*.B.app[0].display-name = "B-A"
*.B.app[0].io.destAddress = "A"
*.B.app[0].io.destPort = 1000 
*.B.app[*].source.packetNameFormat = "%M-%m-%c"
*.B.app[0].source.packetLength = 1200B 
*.B.app[0].source.productionInterval = 490000ns  

# server applications
*.A.numApps = 1
*.A.app[*].typename = "UdpSinkApp"
*.A.app[0].display-name = "B-A"
*.A.app[0].io.localPort = 1000 
```

bugtest.ned :
```
import inet.networks.base.TsnNetworkBase;
import inet.node.ethernet.EthernetLink;
import inet.node.tsn.TsnDevice;
import inet.node.tsn.TsnSwitch;

network bugtest extends TsnNetworkBase
{
    submodules:

        switch: TsnSwitch {
            @display("p=650,400");
        }
        A: TsnDevice {
            @display("p=496,401");
        }
        B: TsnDevice {
            @display("p=650,254");
        }
        C: TsnDevice {
            @display("p=818,401");
        }

    connections:
        switch.ethg++ <--> EthernetLink {  delay = 200ns; } <--> A.ethg++;
        switch.ethg++ <--> EthernetLink {  delay = 200ns; } <--> B.ethg++;
        switch.ethg++ <--> EthernetLink {  delay = 200ns; } <--> C.ethg++;
}
```
Packet A-to-B going the correct path:
<img width="372" alt="A-to-B Packet" src="https://github.com/omnetpp/omnetpp/assets/30881021/e9c03bf1-290d-4d49-9e70-85e1db77baa5">
Packet B-to-A not being able to be correctly routed:
<img width="364" alt="B-to-A Packet" src="https://github.com/omnetpp/omnetpp/assets/30881021/4f51891a-9394-4eb9-b6fa-dd08c1ac9540">

**Comment by rhornig** [comment]
This is the simulation framework's bugtracker. You should have commented/reopened the issue in the INET issue tracker. As far as I see, that particular commit was not included in either 4.5.1 or 4.5.2 so it's available only on the master branch. (I'm not sure why it was not backported to the 4.5.x branch).

**Comment by lukefadedaway** [comment]
Oh yeah sorry, I had multiple GitHub issue tracker tabs open and chose the wrong one
